<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPanel</class>
 <widget class="QDialog" name="AdminPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="logoImage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>251</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>431</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Name: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="nameOutput">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="userLabel">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Username: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="userOutput">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="passwordLabel">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="passwordOutput">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="gradeLabel">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Grade: </string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="gradeOutput">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="hoursLabel">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Hour(s): </string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="hoursOutput">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="eventButton">
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>View Events</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QPushButton" name="editButton">
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>291</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="memberLabel">
      <property name="font">
       <font>
        <family>Kenyan Coffee</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Members</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="logoutButton">
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="studentDropDown"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="registerButton">
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Register Member</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="settingButton">
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
